tomcat/src/main/java/com/techcourse/http/common/HttpStatus.java

```diff
@@ -0,0 +1,22 @@
+package com.techcourse.http.common;
+
+
+public enum HttpStatus {
+
+    OK(200, "OK"),
+    NOT_FOUND(404, "Not Found"),
+    NO_CONTENT(204, "No Content"),
```

Comment:
사소한 의견이지만 2xx, 4xx 묶어서 보여주면 더 가독성이 좋을것 같아요 👍 
```suggestion
    OK(200, "OK"),
    NO_CONTENT(204, "No Content"),
    NOT_FOUND(404, "Not Found"),
```